package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "everli-design-system-frontend",
-  "version": "0.2.6",
+  "version": "0.2.61",
```

Comment:
is this correct? from `.6` to `.61`? 🤔

Reply:
nope, removed - this was added due to some issues with npm cache on vader side
reverted via https://github.com/everli/design-system-frontend/pull/149/commits/493a0b2bd4c260eec68dd62af5749a0e4c9f036b